### ISSUE TITLE
Consider generating appstream metadata for fonts

### ISSUE THREAD

**Issue by n7s** [silnrsi/smith#37]
https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Fonts.html provides a metadata example file that distributions are starting to use to better classify fonts and make them available to users.  

Longer-term, we should consider if smith can provide such metadata.

**Comment by devosb** [comment]
Agree, this would be good. Until then, there is a partial solution already in place.

For font and other packages on packages.sil.org an AppStream metadata file (aka DEP-11, IIRC) should be generated by the server from the uploaded package. From the example above, I would guess such files would not contain the list of font names, as I don't see how such information could be extracted from the packaging.

When the script on packages.sil.org to generate the AppStream files was written, the author mentioned to me that there were no plans to add AppStream capability to PPAs. If that is still the case, then a fonts PPA could not contain this metadata.

**Comment by n7s** [comment]
What script on PSO are your referring to?  I don't think this needs to be server-side but something that could become part of the tarball target.

**Comment by devosb** [comment]
I don't know the name of the script on PSO. It is the same script that Debian and/or Ubuntu uses. Bugs were found when testing the script on PSO, and patches sent to the upstream developer, who was interested in the fixes.

I agree this does not need to be server side. I merely wish to point out what was currently happening. I realize now when I said "should" above I meant I think the script is being run and generating AppStream metadata, not that the approach of having a script on the server is preferred.

**Comment by n7s** [comment]
OK, thanks for clarifying :+1:
